in robotpkg, realtime_tools is not installed as a catkin package  

--- CMakeLists.txt.orig	2014-10-17 16:12:06.141631000 +0200
+++ CMakeLists.txt	2014-10-17 16:12:30.447490000 +0200
@@ -21,7 +21,8 @@
 
 project(dynamic_graph_bridge)
 
-find_package(catkin REQUIRED COMPONENTS roscpp rospy std_msgs message_generation std_srvs geometry_msgs sensor_msgs tf realtime_tools)
+find_package(catkin REQUIRED COMPONENTS roscpp rospy std_msgs message_generation std_srvs geometry_msgs sensor_msgs tf)
+find_package(realtime_tools)
 find_package(Boost REQUIRED COMPONENTS program_options)
 
 catkin_package(CATKIN_DEPENDS message_runtime)
